ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

```diff
@@ -24,6 +24,7 @@ git config --global credential.helper "store --file ${GIT_CREDS_PATH}"
 echo "https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com" > "${GIT_CREDS_PATH}"
 
 cd "$(mktemp -d)"
-git clone "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
+git clone "https://${GITHUB_TOKEN}@github.com/psturc/e2e-tests.git" .
+git checkout HACBS-988
```

Comment:
this change will be reverted once https://github.com/redhat-appstudio/e2e-tests/pull/188 is merged